docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:9.6
+    environment:
+      POSTGRES_USER: gwells
+      POSTGRES_PASSWORD: test1
+    hostname: db
+    volumes:
+      - ./.tmp/psql-dev:/var/lib/postgresql
+  webapp:
+    image: node:8
+    environment:
+      AXIOS_BASE_URL: http://127.0.0.1:8000/gwells/api/v1/
+      CHOKIDAR_USEPOLLING: "True"
+    command: /bin/bash -c "
+        set -x &&
+        cd /app/frontend &&
+        npm install &&
+        npm run build &&
+        npm run ${NPM_CMD:-dev}"
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./app:/app
```

Comment:
i would still like node_modules to live on its own inside the container filesystem so that npm install doesn't affect the local `node_modules`. Some of them aren't cross platform.

Reply:
@stephenhillier You'd made a good point about this earlier.  It's added back in.